damnit/ctxsupport/ctxrunner.py

```diff
@@ -34,13 +34,14 @@
 
 log = logging.getLogger(__name__)
 
+THUMBNAIL_SIZE = 35 # px
```

Comment:
This was already the size we displayed thumbnails, it's just doing all the scaling before we store them. I think quality should actually be slightly better by scaling it all in one go rather than in two. In any case, they look OK to me:

![image](https://github.com/European-XFEL/DAMNIT/assets/327925/fcbd7564-9f6a-4f96-b7b7-aeb5ac2fe592)

Reply:
The same thumbnails before:

![image](https://github.com/European-XFEL/DAMNIT/assets/327925/c24f4c50-6116-49eb-a3e8-a52378f72cef)

Not sure why the colours have changed. :confused:

Reply:
Interesting :laughing: Probably some scaling artifact, but I think that's fine.